src/rapidata/rapidata_client/filter/campaign_filter.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any
+from rapidata.rapidata_client.filter.base_filter import Filter
+from rapidata.api_client.models.campaign_user_filter_model import (
+    CampaignUserFilterModel,
+)
+
+
+class CampaignFilter(Filter):
+
+    def __init__(self, campaign_ids: list[str]):
+        self.campaign_ids = campaign_ids
+
+    def to_model(self) -> Any:
```

Comment:
Not sure if it's necessary to provide the `Any` here. It might overwrite the return type to Any, even if it knows what kind of model it is returning. Same for the other child classes

Reply:
Ah okay. I just copied that from somewhere else